frontend/src/components/UI/Form/ProjectSelectWithSummaryCard/ProjectSelectWithSummaryCard.test.js

```diff
@@ -1,16 +1,17 @@
 import { render, fireEvent } from "@testing-library/react";
 import ProjectSelectWithSummaryCard from "./ProjectSelectWithSummaryCard";
+import { vi } from "vitest";
 
 describe("ProjectSelect", () => {
     const researchProjects = [
         { id: 1, title: "Project 1", description: "Description 1" },
         { id: 2, title: "Project 2", description: "Description 2" },
         { id: 3, title: "Project 3", description: "Description 3" }
     ];
-    const mockClearFunction = jest.fn();
+    const mockClearFunction = vi.fn();
```

Comment:
As I was reviewing this I picked up on something..
A lot of the changes were simply swapping out our test runner (`jest.fn()` --> `vi.fn()` )
Would it make sense for us to set that in something like the `TestApplicationContext.js` in the helpers - something like `mockFn: vi.fn()`, and then reference that anytime we need to define a mock function. This would still require the global change in the tests, like what was done here going from jest to vi - but any future change, could be changed in a single location (ApplicationContext) 🤔 
```javascript
class TestApplicationContext {
    static #helpers = {
        callBackend: vi.fn(),
        authConfig,
        backEndConfig,
        mockFn: vi.fn()
    };
```
Usage:
``` javascript
const mockClearFunction = TestApplicationContext.helpers().mockFn()
```